exporter/azuredataexplorerexporter/adx_exporter_test.go

```diff
@@ -35,16 +35,29 @@ import (
 func TestNewExporter(t *testing.T) {
 	logger := zaptest.NewLogger(t)
 	c := Config{ClusterName: "https://CLUSTER.kusto.windows.net",
-		ClientId:       "unknown",
-		ClientSecret:   "unknown",
-		TenantId:       "unknown",
-		Database:       "not-configured",
-		RawMetricTable: "not-configured",
-		RawLogTable:    "RawLogs",
+		ClientId:               "unknown",
+		ClientSecret:           "unknown",
+		TenantId:               "unknown",
+		Database:               "not-configured",
+		OTELMetricTable:        "not-configured",
+		OTELLogTable:           "RawLogs",
```

Comment:
Make table names consistent , either all of them as not-configured or all of them as OTEL*